exercises/forth/canonical-data.json

```diff
@@ -424,25 +598,67 @@
               "foo"
             ]
           },
-          "expected": [11]
+          "expected": [
+            11
+          ]
         },
         {
           "uuid": "35958cee-a976-4a0f-9378-f678518fa322",
           "description": "cannot redefine numbers",
           "property": "evaluate",
           "input": {
-            "instructions": [": 1 2 ;"]
+            "instructions": [
+              ": 1 2 ;"
+            ]
           },
-          "expected": {"error": "illegal operation"}
+          "expected": {
+            "error": "illegal operation"
+          }
         },
         {
           "uuid": "5180f261-89dd-491e-b230-62737e09806f",
           "description": "errors if executing a non-existent word",
           "property": "evaluate",
           "input": {
-            "instructions": ["foo"]
+            "instructions": [
+              "foo"
+            ]
           },
-          "expected": {"error": "undefined operation"}
+          "expected": {
+            "error": "undefined operation"
+          }
+        },
+        {
+          "uuid": "3c8bfef3-edbb-49c1-9993-21d4030043cb",
+          "description": "only defines locally",
+          "comments": [
+            "This test is meant to determine if instructions that define      ",
+            "words are executed in the state of the program or locally. For   ",
+            "example, in OOP-based approaches, the words should be defined in ",
+            "the same scope, or local to the instance the instructions were   ",
+            "evaluated in.                                                    "
+          ],
+          "scenarios": [
+            "local-scope"
+          ],
+          "property": "evaluate",
+          "input": {
+            "instructionsFirst": [
```

Comment:
I do not believe we have any other instances of canonical data where two test cases with the same `property` have different keys in their `input`. Before, it the case that same `property` implies that `input` contains the same keys for all instances of that `property`. However, I have looked for any documentation of this fact, and found none in README.md or any other place I thought to look.

This would have been a bigger problem before we had an `input` object, but it may not be an issue anymore, since generators now can know to look in a well-defined place (the `input` object) for the presence or absence of any keys that they might be expecting.

Therefore, you'll not get an objection from me if you proceed with this, but you would have gotten one before we had an `input` object. You'll get to be the one breaking new ground.

Reply:
I didn't know how to do this in any other way, but based it on "similar" test cases where we have thingA and thingB. 

Do you think this is an _okay enough_ aproach?

Reply:
Oh, maybe we do: `evaluateBothAreEqual` ?

Reply:
I like the separate property 👍

Reply:
I think I do too. Not overloading any meaning